app/src/components/Common/TimeInput/TimeInput.tsx

```diff
@@ -7,33 +7,64 @@ interface IProps {
   onChange: (value: EditTime) => void;
 }
 
-export const TimeInput = ({ value, onChange }: IProps): JSX.Element => {
+export const TimeInput = ({ value, onChange }: IProps) => {
   const hour = value[0];
   const minute = value[1];
 
-  const updateHour = (value: string) => {
-    onChange([value, minute]);
+  const updateHour = (newHour: string) => {
+    onChange([newHour, minute]);
   };
 
-  const updateMinute = (value: string) => {
-    onChange([hour, value]);
+  const updateMinute = (newMinute: string) => {
+    onChange([hour, newMinute]);
+  };
+
+  const { focusMinuteRefWhenHourFull, minuteRef } = useHourMinuteFocus(hour);
+
+  const selectText = (e: React.FocusEvent<HTMLInputElement>) => {
+    e.target.select();
   };
 
   return (
     <div className={style.timeInputContainer}>
       <input
         className={style.timeInput}
         type="number"
-        onChange={v => updateHour(v.target.value)}
         value={hour}
+        onChange={v => updateHour(v.target.value)}
+        onKeyDown={focusMinuteRefWhenHourFull}
+        onFocus={selectText}
       />
       <div className={style.text}>:</div>
       <input
         className={style.timeInput}
+        ref={minuteRef}
         type="number"
-        onChange={v => updateMinute(v.target.value)}
         value={minute}
+        onChange={v => updateMinute(v.target.value)}
+        onFocus={selectText}
       />
     </div>
   );
 };
+
+const useHourMinuteFocus = (hour: string) => {
```

Comment:
Smud! 🌊 Nå kan man skrive inn så mange tall man vil i minutt, kan føles litt rart når man kun får skrive to i timefeltet

Reply:
Good catch

Reply:
Har ingen god måte å fikse dette på da